solvexia_sdk/api.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+import requests
+import json
+import sys
+
+
+class solvexia_client: 
+    def __init__(self, clientId, clientSecret, env):
```

Comment:
ClientId and ClientSecret are quite sensitive data and ideally should not be passed to your SDK as a parameter. There are different ways to help the users of your SDK to protect their credentials. One way is to allow them to supply a path to the file (json) that contains credentials something like that

```
{ clientId: "", clientSecret: "" }
```

This way they can control access to this file and make sure it's secured. Another way is to use env variables.

Reply:
fixed